packages/expo-cli/src/commands/client/index.js

```diff
@@ -59,6 +60,13 @@ export default program => {
           : 'No custom configuration file could be found. You will need to provide a json file with a valid ios.config.googleMapsApiKey field.';
         disabledServices.googleMaps = { name: 'Google Maps', reason: disabledReason };
       }
+      if (_.has(exp, 'ios.googleServicesFile')) {
+        const contents = await fs.readFile(
+          path.resolve(projectDir, exp.ios.googleServicesFile),
+          'base64'
+        );
+        exp.ios.googleServicesFile = contents;
```

Comment:
Would it make sense to name this property `googleServicesFileBase64`, since we have a Base64-encoded version of the file under it?

Reply:
on android when creating manifest from app json file path is replaced with file content, naming filed in app json `googleServicesFileBase64` would be confusing for users and using two separate fields seems unnecessary.

Reply:
so `googleServicesFile` name is shared between `app.json` and manifest, where in one it. means `path to the file` and in the other one `base64-encoded file contents`, lgtm